### DISCUSSION TITLE
T38

### DISCUSSION THREAD

**Discussion by himikotakamashi** [community/community#66958] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Somebody please tell me how to decode a T38 facsimile image data from a captured pcap file ??

**Comment by Akash1134** [comment]
Hi @himikotakamashi ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
